src/views/datasets/DatasetDetailView.vue

```diff
@@ -229,23 +242,47 @@ watch(
       >
         <div class="datagouv-components" v-if="selectedTabIndex === 0">
           <template v-for="typedResources in resources">
-            <div v-if="typedResources.total" class="fr-mb-4w">
+            <div v-if="typedResources.totalWithoutFilter" class="fr-mb-4w">
               <h2 class="fr-mb-1v subtitle subtitle--uppercase">
                 {{ typedResources.typeLabel }}
               </h2>
-              <ResourceAccordion
-                v-for="resource in typedResources.resources"
-                :datasetId="datasetId"
-                :resource="resource"
-              />
-              <Pagination
-                class="fr-mt-3w"
-                v-if="typedResources.total > pageSize"
-                :page="typedResources.currentPage"
-                :page-size="pageSize"
-                :total-results="typedResources.total"
-                @change="(page) => changePage(typedResources.typeId, page)"
+              <DsfrSearchBar
+                button-text="Rechercher"
+                placeholder="Rechercher"
+                :large="false"
+                class="search-bar"
+                @search="() => doSearch(typedResources.typeId)"
+                @update:modelValue="
+                  (value) => updateQuery(value, typedResources.typeId)
+                "
+                v-if="
+                  typedResources.typeId != 'documentation' &&
```

Comment:
Est-ce qu'on veut garder cette exclusion ?

Reply:
what do you say @geoffreyaldebert? il y avait une raison particulière ?

Reply:
Allez j'enlève le filtre